documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -41,9 +42,28 @@ export default {
 
 ## Options
 
+### config
+
+Path to your [Wrangler configuration file](https://developers.cloudflare.com/workers/wrangler/configuration/). If you would like to use a Wrangler configuration filename other than `wrangler.jsonc`, you can specify it using this option.
```

Comment:
You are using `unstable_readConfig` to read the config file right? if that's the case by default `wrangler.json` and `wrangler.toml` are picked up as well, should those also be included in this sentence?